.gitignore

```diff
@@ -35,6 +35,7 @@ out/
 restlicsharpdata/Debug
 restlicsharpdata/obj
 restlicsharpdata/bin
+restlicsharpdata/restlicsharpdata.csproj
```

Comment:
This should not be ignored.

We should add the following in ignore list

`### Gradle ###`
.gradle
**/build/

`### IntelliJ ###`
*.iml
*.ipr
*.iws
/out/
.idea/